src/Microsoft.Extensions.Configuration.Abstractions/IConfigurationSection.cs

```diff
@@ -1,13 +1,27 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-
 namespace Microsoft.Extensions.Configuration
 {
+    /// <summary>
+    /// REVIEW: Really not confident these are correct.
+    /// Represents a sub-section of application configuration values.
+    /// </summary>
     public interface IConfigurationSection : IConfiguration
```

Comment:
Thanks @lodejard. Can you help out with `IConfigurationProvider.GetChildKeys` too please?